src/data/history.ts

```diff
@@ -457,7 +457,7 @@ export const getStatisticIds = (
   statistic_type?: "mean" | "sum"
 ) =>
   hass.callWS<StatisticsMetaData[]>({
-    type: "history/list_statistic_ids",
+    type: "recorder/list_statistic_ids",
```

Comment:
Changing the domain means it should also move to `data/recorder.ts` instead of `data/history.ts`

Reply:
That's right, but all other statistics related stuff, which is already accessed via the recorder domain is in `data/history.ts`
Can we move that in a separate PR?

Reply:
ok